python/paddle/fluid/dygraph/__init__.py

```diff
@@ -46,7 +47,7 @@
 __all__ += base.__all__
 __all__ += nn.__all__
 __all__ += tracer.__all__
-__all__ += profiler.__all__
+# __all__ += profiler.__all__
```

Comment:
how about just delete it instead of comment it off

Reply:
done